packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1915,6 +1915,7 @@ export default mixins(
 				newNodeData.webhookId = uuid();
 			}
 
+			newNodeData.pristine = true;
```

Comment:
this approach is problematic because it's updating the workflow json and its getting saved to the database and copied when workflow is copied.. since it's just a ui flag for a little interaction, does not make sense to store this.. 

I would recommend you use `nodeMetadata` in the workflows store which is built to avoid this exact problem

Reply:
That's a good point, I've changed it to use `nodeMetadata`. Thanks!